plugins/utils/dashboard.nix

```diff
@@ -95,46 +131,145 @@ in {
         default = {};
       };
 
-      hideStatusline = mkOption {
-        description = "Whether to hide statusline in Dashboard buffer";
+      changeToVcsRoot = mkOption {
+        type = types.nullOr types.bool;
+        description = "For open file in hyper mru. It will change to the root of vcs";
+        default = null;
+      };
+
+      disableMove = mkOption {
+        type = types.nullOr types.bool;
+        description = "Disable the move keymap for hyper";
+        default = null;
+      };
+
+      hide = {
+        statusline = mkOption {
+          description = "Whether to hide statusline in Dashboard buffer";
+          type = types.nullOr types.bool;
+          default = null;
+        };
+
+        tabline = mkOption {
+          description = "Whether to hide tabline in Dashboard buffer";
+          type = types.nullOr types.bool;
+          default = null;
+        };
+
+        winbar = mkOption {
+          description = "Whether to hide winbar in Dashboard buffer";
+          type = types.nullOr types.bool;
+          default = null;
+        };
+      };
+
+      packages.enable = mkOption {
+        description = "Whether to display installed Vim packages";
         type = types.nullOr types.bool;
         default = null;
       };
 
-      hideTabline = mkOption {
-        description = "Whether to hide tabline in Dashboard buffer";
+      week_header.enable = mkOption {
```

Comment:
Please use `weekHeader` for the "old" kind of options. Plugin defined case is only used in the new mkNeovimPlugin implementations.